### PR TITLE
feat: add manage-scripts to switch system modules value

### LINKED ISSUE #6525: default value of ticket_include is false in system
**Describe the bug**
ticket_include is set to 'false' by default which complicates setting up a new event for testers in the development

**To Reproduce**
1. Go to createdb.py file
2. Set the default value to true

**Expected behavior**
A tester should directly be able to create event while testing without enabling it manually in the database

**Additional context**
I am taking up this issue

### PR DESCRIPTION
**Pull request by akshat0047**
Fixes #6525  @prateekj117 @iamareebjamal 

#### Short description of what this resolves:
This adds a manage-script which helps the tester to switch any system module On/Off through command line

#### Changes proposed in this pull request:

- Added a manage command with the name switch 
- switch command provides the option to switch a specific module or all of them by specifying module_name or all
- Updated documentation in readme.md under the 'Testing' heading

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,18 +289,21 @@ The tentative release policy, for now (since there is a lot of activity and a lo
...
+
+-   Before you join development, please set up the system on your local machine and go through the application completely. Press on any link/button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
+-   Test the application on your machine and explore the admin area. The test deployment on Heroku will not give you access to the admin section, where you can switch on/off modules, e.g. ticketing and add keys for services, e.g. storage on S3.
+-   If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
```
Comment (iamareebjamal) [inline README.md]: All changes in README are unrelated to the open issue. Please stick to the changes proposed in the PR. Open another issue of you want to fix other things

File: manage.py [hunk 2]
```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def switch(system):
```
Comment (iamareebjamal) [inline manage.py]: See the option below. Rename to module and pass name and on/off as parameters

File: manage.py [hunk 3]
```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
...
+        for key in keys:
+            value = getattr(Module.query.first(), key)
+            time.sleep(1)
+            setattr(Module.query.first(), key, not value)
+            db.session.commit()
+            print("Module %s turned %s" % (key, convey[str(not value)]))
+    else:
+        if system in keys:
+            value = getattr(Module.query.first(),system)
+            time.sleep(1)
```
Comment (iamareebjamal) [inline manage.py]: Is the system so fast that we deliberately need to make it slow? Or is it needed for some reason

File: manage.py (hunk 2 above)
Comment (akshat0047) [reply manage.py]: ok, on it

File: manage.py (hunk 3 above)
Comment (akshat0047) [reply manage.py]: I apologize for pushing this as it was making sense with some previous functionality here, similar mistake may not be repeated.

File: README.md (hunk 1 above)
Comment (akshat0047) [reply README.md]: I didn't change this, must be a result of some rebase/squashing/stashing, fixing it

Comment (akshat0047) [reply README.md]: Fixed

File: manage.py (hunk 2 above)
Comment (akshat0047) [reply manage.py]: Implemented, please review

File: manage.py (hunk 3 above)
Comment (akshat0047) [reply manage.py]: Removed

File: manage.py [hunk 4]
```diff
@@ -36,6 +37,27 @@ def add_event_identifier():
...
+    if switch not in ['on', 'off']:
+        print("Choose either state On/Off")
+
+    elif name == 'all':
+        for key in keys:
+            setattr(Module.query.first(), key, convey[switch])
+            db.session.commit()
+            print("Module %s turned %s" % (key, switch))
+    else:
+        if name in keys:
```
Comment (iamareebjamal) [inline manage.py]: Can be changed to elif

Comment (akshat0047) [reply manage.py]: Implemented

Comment (akshat0047) [reply manage.py]: Updating...

Comment (akshat0047) [reply manage.py]: Updated, consistent with upstream.

File: manage.py [hunk 5]
```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
...
+    if switch not in ['on', 'off']:
+        print("Choose either state On/Off")
+
+    elif name == 'all':
+        for key in keys:
+            setattr(Module.query.first(), key, convey[switch])
+            db.session.commit()
+            print("Module %s turned %s" % (key, switch))
+    elif name in keys:
+            setattr(Module.query.first(), name, convey[switch])
```
Comment (prateekj117) [inline manage.py]: @akshat0047 Fix indentation here. Make it four spaces as used in other parts of the file.

File: manage.py [hunk 6]
```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
...
+            db.session.commit()
+            print("Module %s turned %s" % (name, switch))
+    else:
+        print("Invalid module selected")
+
```
Comment (prateekj117) [inline manage.py]: PEP 8: expected 2 blank lines, found 1.

File: manage.py (hunk 5 above)
Comment (akshat0047) [reply manage.py]: Done

File: manage.py (hunk 6 above)
Comment (akshat0047) [reply manage.py]: Done

File: manage.py [hunk 7]
```diff
@@ -36,6 +37,27 @@ def add_event_identifier():
...
+def module(name, switch):
+    keys = [i.name for i in Module.__table__.columns][1:]
+    convey = {"on": True, "off": False}
+    if switch not in ['on', 'off']:
+        print("Choose either state On/Off")
+
+    elif name == 'all':
+        for key in keys:
+            setattr(Module.query.first(), key, convey[switch])
+            db.session.commit()
```
Comment (iamareebjamal) [inline manage.py]: Move commit to last line instead of commiting in each iteration

Comment (akshat0047) [reply manage.py]: Implemented
